examples/podtatohead-deployment/keptnevaluationprovider.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: lifecycle.keptn.sh/v1alpha1
+kind: KeptnEvaluationProvider
+metadata:
+  name: prometheus
+  namespace: podtato-kubectl
+spec:
+  targetServer: "http://prometheus-k8s.monitoring.svc.cluster.local:9090" #string
+  secretName: prometheusLoginCredentials #secret name, optional
```

Comment:
do we need a secret in this case?

Reply:
removed the example, as it should be part of another PR